query_generators/anthropic.py

```diff
@@ -48,11 +48,8 @@ def get_completion(
     ):
         """Get Anthropic chat completion for a given prompt and model"""
         generated_text = ""
-        sys_prompt = prompt.split("### Input:")[0]
-        user_prompt = prompt.split("### Input:")[1].split("### Response:")[0]
-        messages = [
-            {"role": "user", "content": [{"type": "text", "text": user_prompt}]}
-        ]
+        prompt
```

Comment:
were you intending to do something here?

Reply:
ah whoops was a typo – fixed now, thanks!